File: ex3_conversao_temperaturas/main.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    double celsius;

    cout << "Temperatura em °C: ";
    cin >> celsius;

    double fahrenheit = (celsius * 9.0 / 5.0) + 32.0;

    cout << fixed << setprecision(2);
    cout << celsius << "°C equivale a " << fahrenheit << "°F" << endl;

    return 0;
}

File: ex3_conversao_temperaturas/main.py
celsius = float(input("Temperatura em °C: "))

# Fórmula: multiplicação e divisão têm mesma precedência, avaliadas da esquerda para direita
# Colocamos parênteses para deixar explícito que (celsisus * 9 / 5) é feito antes de somar 32
fahrenheit = (celsius * 9 / 5) + 32

print(f"{celsius}°C equivale a {fahrenheit:.2f}°F")